policies/connectivity/Pod.yaml

```diff
@@ -79,8 +105,8 @@ spec:
                   [[- end ]]
                   [[- if $proxy.noProxy ]]
                   - name: NO_PROXY
-                    value: [[ join "," $proxy.noProxy ]]
+                    value: [[`"[[ fullNoProxy ]]"`]]
                   - name: no_proxy
-                    value: [[ join "," $proxy.noProxy ]]
+                    value: [[`"[[ fullNoProxy ]]"`]]
                   [[- end ]]
```

Comment:
Why do we need a copy of the same thing with square brackets?

Reply:
I asked the same question in slack. I didn't get why do we have a template for a template :(